### ISSUE TITLE
Cerebral infarct

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7043]
**Preferred term label:**

Cerebral infarct

**Synonyms**

Brain infarct, Cerebrum infarct

**Definition (free text, please give PubMed ID)**

A necrotic lesion in the cerebrum resulting from a sudden insufficiency of arterial or venous blood supply due to emboli, thrombi or mechanical factors. 
PMID:10867785

**Parent term (use hpo.jax.org/app)**
abnormal cerebral morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Alzheimer's Disease (OMIM:104300)

**Your nano-attribution (ORCID)**

https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
@sarahstofel I think this would be good under the existing Stroke hierarchy, e.g., MCA stroke.

https://hpo.jax.org/app/browse/term/HP:0001297

**Comment by pnrobinson** [comment]
Thanks, adding as HP_0025722
I think that  in the future we need to reorganize the stroke terms, they should refer to the functional deficit, and the 'infarct' terms are actually the morphological anomaly.
